src/kubernetes-tool/templates/app.vue

```diff
@@ -20,25 +20,41 @@ limitations under the License.
             <SplashScreen @result="resultSet" />
         </div>
         <div v-else>
-            <Header :title="i18n.templates.app.title">
-                <template v-slot:description>
-                </template>
-                <template v-slot:header>
-                </template>
-                <template v-slot:buttons>
-                    <a class="button" @click="mainMenu">{{ i18n.templates.shared.mainMenu }}</a>
-                </template>
-            </Header>
+            <div class="columns">
+                <div class="column is-2" style="height: 100%; top: 0; position: sticky;">
```

Comment:
This does not work on small screens at all. Need to make this collapsible. If there isn't space for it to fit side-by-side, it should float on top and be collapsed by default.

1280x800:
![image](https://user-images.githubusercontent.com/12371363/72379227-ed615100-370a-11ea-8540-ffc0ee5699ef.png)

Reply:
Yeah, I forgot to mention this bug. I feel like it should wait until any designs from @AmNotADev (although I'd like your opinion on this)

Reply:
The plan was to get this merged and then for Raven to do their UI review whenever they have time and create issues on the repo for changes. As such, do you want to just make a minimum fix for this and we can revise the design later?

Reply:
Ok, shall do!